pkg/kubectl/scale.go

```diff
@@ -155,24 +158,27 @@ type ReplicationControllerScaler struct {
 	c client.Interface
 }
 
-func (scaler *ReplicationControllerScaler) ScaleSimple(namespace, name string, preconditions *ScalePrecondition, newSize uint) error {
+// ScaleSimple does a simple one-shot attempt at scaling. It returns the
+// resourceVersion of the replication controller if the update is successful.
+func (scaler *ReplicationControllerScaler) ScaleSimple(namespace, name string, preconditions *ScalePrecondition, newSize uint) (string, error) {
```

Comment:
@wojtek-t I removed the name of the output parameter and added the comment. PTAL. Thanks.